tests/regression/router-sanitizer.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+source tests/regression/common.sh
+
+check_env
+setup_sanitizer_opts
+
+# Get command line arguments
+# Current arguments parsed are <sleep_time> <remaining_args>
+get_cli_args $@
+
+# Install prerequisites
+make
+pip install --user -r tests/regression/requirements.txt
+redis-server &
+
+# Iterate over all available build options
+for (( i = 0; i < ${#SAN_OPTIONS[@]}; i++ )); do
+    option=${SAN_OPTIONS[${i}]}
+
+    bazel run accelerator ${option} -c dbg -- --ta_port=${TA_PORT} &
+    TA=$!
+    sleep ${sleep_time} # TA takes time to be built
```

Comment:
Can you avoid using `sleep`?

Reply:
Now the bash script will wait for the signal from tangle-accelerator